pkg/Elmah.AspNet.nuspec

```diff
@@ -2,17 +2,16 @@
 <package xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
         <id>elmah.aspnet</id>
-        <version>2.0.0-alpha001</version>
-        <title>ELMAH AspNet Library</title>
+        <version>2.0.0-alpha002</version>
```

Comment:
What did you have in mind with this pre-release version bump?

Reply:
Adjusted